### PR TITLE
16th Bug Crusade

### PR DESCRIPTION
**Pull request by zumbak04**
Changelog:
- Fixed issue where some death types had no localization.
- Fixed issue where undead, demons and other already interfile creatures like harpies could get Interfile traits.
- Fixed issue where Legendary Druid trait didn't give health bonus.
- Fixed issue where the reviving of the Black Empire didn't copy laws of your former primal title.
- Removed Cenarius' nickname because his full name sounded like "Forest Lord Cenarius the Lord of the Forest".
- Fixed issue where sayaadi could be spawned with both opposite traits like cruel and kind or 2 lifestyles.
- Fixed issue where c'traxxi, spawned in WCVOD.10015, could have negative genetic traits.
- Fixed issue where future leader of Theramore couldn't invite non-independent ruler to said to Kalimdor.
- Fixed issue where settlers like the Lich King, leader of the New Horde, Theramore etc. didn't declare war for all titles they had claims on.
- Fixed issue where settlers like the Lich King, leader of the New Horde, Theramore etc. upgraded tribals of players and created free temples, cities and castles.
- Fixed issue where AI didn't care about opinion in WCDIP.10 and WCDIP.15.
- Fixed localization of "The Ancient Creature" event chain (starting from WCVOD.10005 event).
- Fixed issue where barons of Arcane satyrs in Northrend spawned spouses non-stop, but couldn't marry them.
- Adapted WoL.10114 event. It shows name of mounts (horses, raptors, kodo etc.) depending on culture.
- Adapted WoL.10210 event. Bishop forbids you to build high towers only if religion has hell/heaven concept (Odyn, Holy Light, Scarlet Light etc.).
- Adapted WoL.10302 event. No longer can buy religious statues if your religion is secular (Titanic, Arcane, Nerubian etc.).
- Adapted WoL.10303 event. No longer can build wine fountain if you can't drink wine (Undead, Demon, Void Being, Ancient).
- Fixed issue where using "Heal Wounds" decision on yourself didn't spend Mana.
- Invader titles like the Scourge, Burning Legion, Black Empire can't be claimed and usurped. So, Claim All CB is available against the Scourge, Burning Legion, Black Empire.

Secret Changelog:
- Fixed issue when the game tried to add trait already having opposite of it.
- Fixed issue when the game tried to add trait already having it.
- Сommented out Major Epidemics game rule since it's not used yet.

### CODE REVIEW

**Comment by geffe1** [comment]
>Fixed issue where barons of Arcane satyrs in Northrend spawned spouses non-stop, but couldn't marry them.

No idea how this would even be possible, as they do not have any vassals that can marry due to being Satyr- however I will test it by taking over some land and forcing a troll to take it and see if this still occurs.

**Comment by geffe1** [comment]
>Fixed issue where using "Heal Wounds" decision on yourself didn't spend Mana.

For me it won't drain any mana still. Healed my wounds, said I'd lose 1 mana and still had 3 afterwards.

**Comment by MemeWizard** [comment]
>Fixed issue where barons of Arcane satyrs in Northrend spawned spouses non-stop, but couldn't marry them.

Is this somewhat related to appearance of characters, who both married and can't marry?

<details>

image


</details>

>Fixed issue where settlers like the Lich King, leader of the New Horde, Theramore etc. didn't declare war for all titles they had claims on.

Nope?

<details>

image

</details>

<details>

image
image

</details>

**Comment by zumbak04** [comment]
> > Fixed issue where settlers like the Lich King, leader of the New Horde, Theramore etc. didn't declare war for all titles they had claims on.
> 
> Nope?

It's because Claim All CB is blocked against the Scourge, Legion, Black Empire. Making it possible opens ways to possible bugs. There's only one hope called `can_be_claimed` rule, but I'm not sure it will work.

**Comment by zumbak04** [comment]
> It's because Claim All CB is blocked against the Scourge, Legion, Black Empire. Making it possible opens ways to possible bugs. There's only one hope called `can_be_claimed` rule, but I'm not sure it will work.

Implemented. Let's see if it works.

> How about make titular duke-tier Theramore title for exodus leader and change existing Theramore title to something else?
 
This sounds like a suggestion.

> So, it's ok that Cenarius now can get nicknames?

The same. There's a lot of characters beside Cenarius that can get their lore nicknames if they got ones.

File: events/wc_class_events.txt [hunk 1]
```diff
@@ -1478,32 +1473,35 @@ character_event = {
...
-				}
+				
+				# Lifestyle
+				remove_lifestyle_trait_effect = yes
+				add_trait = seductress
+				
+				# Traits
+				add_trait = fair
+				add_trait_silently_lustful_effect = yes
+				remove_trait_silently_chaste_effect = yes
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply events/wc_class_events.txt]: It doesn’t always add lustful. If character spawns with random chaste, it clears it without adding lustful.

File: events/wc_void_events.txt [hunk 2]
```diff
@@ -2525,26 +2534,23 @@ narrative_event = {
...
+			
+			character_event = { id = WCRAC.10 }
+			
+			# Education
+			remove_education_effect = yes
+			add_trait = brilliant_strategist
+			add_trait = class_warrior_7
+			
+			# Traits
+			remove_trait_silently_wroth_effect = yes
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply events/wc_void_events.txt]: It doesn’t always add patient. If character spawns with random wroth, it clears it without adding patient.

**Comment by zumbak04** [comment]
> > Fixed issue where barons of Arcane satyrs in Northrend spawned spouses non-stop, but couldn't marry them.
> 
> Is this somewhat related to appearance of characters, who both married and can't marry?

Fixed!

File: events/wc_class_events.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (MemeWizard) [reply events/wc_class_events.txt]: It's only `add_trait_patient_lustful_effect` problem. You can just replace them both with `add_trait = patient`

File: events/wc_void_events.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (MemeWizard) [reply events/wc_void_events.txt]: It's only `add_trait_patient_lustful_effect` problem. You can just replace them both with `add_trait = patient`

File: events/wc_class_events.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (zumbak04) [reply events/wc_class_events.txt]: It's not a problem. I want it to be kinda random.

[the remark this answers is not in the document]
Comment (MemeWizard) [reply events/wc_class_events.txt]: ok, no problem then

**Pull request merged**